py/selenium/webdriver/remote/webdriver.py

```diff
@@ -1222,3 +1226,112 @@ def delete_downloadable_files(self) -> None:
             raise WebDriverException("You must enable downloads in order to work with downloadable files.")
 
         self.execute(Command.DELETE_DOWNLOADABLE_FILES)
+
+    @property
+    def fedcm(self) -> FedCM:
+        """
+        :Returns:
+            - FedCM: an object providing access to all Federated Credential Management (FedCM) dialog commands.
+
+        :Usage:
+            ::
+
+                title = driver.fedcm.title
+                subtitle = driver.fedcm.subtitle
+                dialog_type = driver.fedcm.dialog_type
+                accounts = driver.fedcm.account_list
+                driver.fedcm.select_account(0)
+                driver.fedcm.click_continue()
+                driver.fedcm.cancel_dialog()
+                driver.fedcm.enable_delay()
+                driver.fedcm.disable_delay()
+                driver.fedcm.reset_cooldown()
+        """
+        return self._fedcm
+
+    @property
+    def supports_fedcm(self) -> bool:
```

Comment:
are we really wanting this to be public?

Reply:
I think it can be used to verify support for FedCM by end users, the `_require_fedcm_support` function can internally check and raise error but having a way to check beforehand is good.